keyboards/xd75/keymaps/boy_314/keymap.c

```diff
@@ -125,6 +125,12 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  }
 };
 
+
+
+const uint16_t PROGMEM fn_actions[] = {
```

Comment:
Please remove this section, `fn_actions` is deprecated (`process_record_user` is the way to go now) and you're not using it anyway.